homeassistant/components/stream/core.py

```diff
@@ -19,20 +19,37 @@
 PROVIDERS = Registry()
 
 
+@attr.s(slots=True)
+class Part:
+    """Represent a segment part."""
+
+    duration: float = attr.ib()
+    independent: bool = attr.ib()
```

Comment:
I'm ambivalent on this. Later we use this to set the "INDEPENDENT" tag on the output, so at some stage there will be a mismatch between the "keyframe" and "independent" terminology. I guess it might slightly cleaner to keep all the code references as "keyframe" and allow that mismatch to occur when we translate it to text, but in terms of code readability it will be the same.

Reply:
Made the change